docs/source/build_app/build-application.rst

```diff
@@ -154,3 +164,9 @@ Static linking, Single-threaded |short_name|:
 
      clang++ -fsycl -DONEAPI_DAAL_USE_MKL_GPU_GEMM my_first_daal_program.cpp -Wl,
      --start-group <install dir>/daal/latest/lib/intel64/libonedal_core.a <install dir>/daal/latest/lib/intel64/libonedal_sequential.a -lpthread -ldl -lOpenCL -foffload-static-lib=<install dir>/daal/latest/lib/intel64/libonedal_sycl.a -Wl,--end-group
+
+Dynamic library versions
+---------------------
+Some dynamic library files and links have major and minor binary versions in names.
+When new library version without ABI compatibility to previous versions are going to be released, major version is incremented and minor is set to 0.
+When new library version with ABI compatibility to previous versions and additional API are going to be released, minor version is incremented.
```

Comment:
Incorrect statements. In fact the library follows semantic versioning instead.

Reply:
Rewritten to align with semantic versioning